api/src/models/ticket_info.rs

```diff
@@ -36,19 +35,13 @@ pub struct TicketInfo {
 }
 
 impl From<Issue> for TicketInfo {
-    fn from(issue: Issue) -> Self {
-        let summary = match issue.summary() {
-            None => "".to_owned(),
-            Some(summary) => summary,
-        };
+    fn from(issues: Issue) -> Self {
```

Comment:
Since we are referring to a single issue, its preferred to call it issue itself.
```suggestion
    fn from(issue: Issue) -> Self {
```